terraform/modules/data_share_service/ecs.tf

```diff
@@ -65,7 +66,7 @@ resource "aws_ecs_task_definition" "gdx_data_share_poc" {
         { "name" : "TASK_ROLE_ARN", "value" : aws_iam_role.ecs_task.arn },
 
         { "name" : "AWS_XRAY_CONTEXT_MISSING", "value" : "IGNORE_ERROR" },
-        { "name" : "ADMIN_ACTION_ALERT_SNS_TOPIC_ARN", "value" : module.sns_admin_alerts.topic_arn },
+        local.admin_action_alert,
```

Comment:
I think I'd be a tiny bit surprised if this behaves, what does a terraform plan do?

Reply:
Huh, looks happy enough from
``` # module.data_share_service.aws_ecs_task_definition.gdx_data_share_poc must be replaced
-/+ resource "aws_ecs_task_definition" "gdx_data_share_poc" {
      ~ arn                      = "arn:aws:ecs:eu-west-2:776473272850:task-definition/dev-gdx-data-share-poc:874" -> (known after apply)
      ~ arn_without_revision     = "arn:aws:ecs:eu-west-2:776473272850:task-definition/dev-gdx-data-share-poc" -> (known after apply)
      ~ container_definitions    = (sensitive value) # forces replacement
      ~ id                       = "dev-gdx-data-share-poc" -> (known after apply)
      ~ revision                 = 874 -> (known after apply)
      - tags                     = {} -> null
        # (9 unchanged attributes hidden)
    }
```
so I guess we'll see :D